==== External Call To User-Supplied Address ====
SWC ID: 107
Severity: Low
Contract: MFI
Function name: claimWrongTokenTransfer(address,uint256)
PC address: 4953
Estimated Gas Usage: 4649 - 74293
A call to a user-supplied address is executed.
An external message call to an address specified by the caller is executed. Note that the callee account might contain arbitrary code and could re-enter any function within this contract. Reentering the contract in an intermediate state may lead to unexpected behaviour. Make sure that no state modifications are executed after this call and/or reentrancy guards are in place.
--------------------
In file: /tmp/b040e8ea512e062812ba11cd7b025c97.sol:207

) = _msgSender().call{value:address(this).bala

--------------------
Initial State:

Account: [CREATOR], balance: 0x1, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [CREATOR], function: claimWrongTokenTransfer(address,uint256), txdata: 0xb5ff3532000000000000000000000000deadbeefdeadbeefdeadbeefdeadbeefdeadbeef0000000000000000000000000000000000000000000000000000000000000001, value: 0x0

==== Multiple Calls in a Single Transaction ====
SWC ID: 113
Severity: Low
Contract: MFI
Function name: claimWrongTokenTransfer(address,uint256)
PC address: 4953
Estimated Gas Usage: 4649 - 74293
Multiple calls are executed in the same transaction.
This call is executed following another call within the same transaction. It is possible that the call never gets executed if a prior call fails permanently. This might be caused intentionally by a malicious callee. If possible, refactor the code such that each transaction only executes one external call or make sure that all callees can be trusted (i.e. they’re part of your own codebase).
--------------------
In file: /tmp/b040e8ea512e062812ba11cd7b025c97.sol:207

) = _msgSender().call{value:address(this).bala

--------------------
Initial State:

Account: [CREATOR], balance: 0x3, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [CREATOR], function: claimWrongTokenTransfer(address,uint256), txdata: 0xb5ff353200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001, value: 0x0

==== External Call To User-Supplied Address ====
SWC ID: 107
Severity: Low
Contract: MFI
Function name: claimWrongTokenTransfer(address,uint256)
PC address: 5138
Estimated Gas Usage: 4649 - 74293
A call to a user-supplied address is executed.
An external message call to an address specified by the caller is executed. Note that the callee account might contain arbitrary code and could re-enter any function within this contract. Reentering the contract in an intermediate state may lead to unexpected behaviour. Make sure that no state modifications are executed after this call and/or reentrancy guards are in place.
--------------------
In file: /tmp/b040e8ea512e062812ba11cd7b025c97.sol:208

    require(result, "MFI: ETH Transfer Failed");
    }

--------------------
Initial State:

Account: [CREATOR], balance: 0x1, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [CREATOR], function: claimWrongTokenTransfer(address,uint256), txdata: 0xb5ff3532000000000000000000000000deadbeefdeadbeefdeadbeefdeadbeefdeadbeef0000000000000000000000000000000000000000000000000000000000000001, value: 0x0

==== Multiple Calls in a Single Transaction ====
SWC ID: 113
Severity: Low
Contract: MFI
Function name: claimWrongTokenTransfer(address,uint256)
PC address: 5138
Estimated Gas Usage: 4558 - 74016
Multiple calls are executed in the same transaction.
This call is executed following another call within the same transaction. It is possible that the call never gets executed if a prior call fails permanently. This might be caused intentionally by a malicious callee. If possible, refactor the code such that each transaction only executes one external call or make sure that all callees can be trusted (i.e. they’re part of your own codebase).
--------------------
In file: /tmp/b040e8ea512e062812ba11cd7b025c97.sol:208

    require(result, "MFI: ETH Transfer Failed");
    }

--------------------
Initial State:

Account: [CREATOR], balance: 0x80000080030, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [CREATOR], function: claimWrongTokenTransfer(address,uint256), txdata: 0xb5ff353200000000000000000000000000000000000000000000000000000000000000000101010101010101010101010000000000000000000000000000000000000000, value: 0x0